chesster.js

```diff
@@ -644,6 +647,78 @@ controller.hears([
     });
 });
 
+controller.hears([
+    'pairing ([a-zA-Z0-9]+) ([1-8])'
+], [
+    'direct_mention',
+    'direct_message'
+], function(bot, message) {
+    bot_exception_handler(bot, message, function() {
+        var player_name = message.match[1];
+        var round = message.match[2]
+        preparePairingCompetitorMessage(player_name, round, function(response) {
+            bot.reply(message, response);
+        });
+    });
+});
+
+function preparePairingCompetitorMessage(player, round, callback) {
+    var self = this;
+    loadSheet(self, function() {
+
+        for (var i = 0; i < self.rounds.length; i++) {
+            if (self.rounds[i].title === "Round " + round) {
+                // Get the spreadsheet for that round
+                parsePairingForRound(self.rounds[i], player, callback);
+                return;
+            } 
+        }
+
+        // If the round has not yet been scheduled, reply that and exit
+        callback("Round " + round + " has not been scheduled yet");
+    });
+}
+
+function parsePairingForRound(roundSheet, player, callback) {
```

Comment:
We have a "find_pairing"  method in spreadsheet.js - I'd like to see this use that, if possible. If that one is lacking - we should upgrade it so it supports both functionality.

In fact, it would be awesome if we were supporting finding pairings via only one side as that might help us locate and update schedules / results for people that played with alternates.

Any chance you can look at merging the two methods into one?